Spring.FluentContext/IFluentConfigurator.cs

```diff
@@ -0,0 +1,7 @@
+namespace Spring.FluentContext
+{
+    public interface IFluentConfigurator
```

Comment:
What is the purpose of this interface?
I do not see any usage of it. Was it a mistake to add it here?